src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ExclusionZoneService.kt

```diff
@@ -146,15 +146,15 @@ class ExclusionZoneService(
     var pdfDoc: PDDocument? = null
     try {
       pdfDoc = PDDocument.load(fileStream)
-      val page = pdfDoc.pages.first()
-      page.resources.xObjectNames.forEach {
-        val o = page.resources.getXObject(it)
-        if (o is PDImageXObject) {
-          val baos = ByteArrayOutputStream()
-          ImageIO.write(o.image, "jpg", baos)
-          return baos.toByteArray()
-        }
-      }
+      val renderer = PDFRenderer(pdfDoc)
+      val firstImage = renderer.renderImage(0)
+      val croppedImage = firstImage.getSubimage(50, 50, firstImage.width - 100, firstImage.height - 100)
+      val baos = ByteArrayOutputStream()
+      ImageIO.write(croppedImage, "jpg", baos)
+
+      // Scale it back up to original size without the borders ?
```

Comment:
Is this a TODO?

Reply:
Just a thought for now - didn't intend to leave it it. It looks ok as is for now, but I think there are a couple of other things we could do to make the map sharper. I've created a ticket and put into the backlog for these.  1) Alter the format from JPEG to PNG (lossless compression) - for maps with lines and small text, this might really help with clarity - depending on the quality of printers in prisons!. 2) Scale the image up slightly so it fills more of the page.  A combination of both would improve it - but I'd rather wait and see what the feedback is?